Makefile

```diff
@@ -34,12 +34,15 @@ init:
 	pip install pipenv --upgrade
 	pipenv run pip install --upgrade pip
 	pipenv install
-	cp {Pipfile,Pipfile.lock,setup.py} ${BUILD_CONTEXT}
+	cp Pipfile ${BUILD_CONTEXT}
+	cp Pipfile.lock ${BUILD_CONTEXT}
+	cp setup.py ${BUILD_CONTEXT}
+
 
 # Builds and moves container python library into the Docker build context
 build-container-library: init
 	python setup.py bdist_wheel;
-	cp dist/*.whl ${BUILD_CONTEXT}
+	cp -- dist/*.whl ${BUILD_CONTEXT}
```

Comment:
Do we need `--` here ?

Reply:
I think this prevents bugs when there's a file with a name which has a dash, it can be interpreted as an option, IIRC is a safe idiom for bash scripts to avoid unintended failures due to a file having a name "-r" for example.